### DISCUSSION TITLE
Error creating database on 5.0.1

### DISCUSSION THREAD

**Discussion by LuisVilaplana** [oqtane/oqtane.framework#3606] [General] [answered]
Hi, I installed Oqtane 5.0.1 source code on a new laptop with Windows 11 Pro,  new Visual Studio 2022 (as administrator) and new SQL Server Developer 2022. When I run it the first time, the setup wizard appears but got this error no matter what database option I choose:

An Error Occurred Creating The Database. This Is Usually Related To Your User Not Having Sufficient Rights To Perform This Operation. Please Note That You Can Also Create The Database Manually Prior To Initiating The Install Wizard. System.ArgumentNullException: Value cannot be null. (Parameter 'type') at System.ArgumentNullException.Throw(String paramName) at System.Activator.CreateInstance(Type type, Boolean nonPublic, Boolean wrapExceptions) at Oqtane.Infrastructure.DatabaseManager.CreateDatabase(InstallConfig install) in C:\Develop\Proyects\ProsoftPOS Octane\oqtane.framework\Oqtane.Server\Infrastructure\DatabaseManager.cs:line 220

Any idea how to solve this?

Other apps connect without problems to this SQL server

**Comment by sbwalker** [comment, marked as answer]
@LuisVilaplana  after you opened the Oqtane.sln file in Visual Studio, did you you rebuild the entire solution (Build / Rebuild Solution)? I suspect the Database Providers do not exist in your installation. You can verify by looking in your Oqtane.Server\bin\Debug\net8.0 folder and check for the existence of the Oqtane.Database.SqlServer.dll assembly.

**Reply by LuisVilaplana** [reply]
Wow, you were right. I had just run a normal build. Thank you so much

**Comment by TheCodeifier** [comment]
Hi, I got some issues with Oqtane and then I came across this topic here and it was helpful for me too. I've succeeded in installing Oqtane for the second time on my second project.
